database/seeds/DatabaseSeeder.php

```diff
@@ -98,6 +100,30 @@ public function run()
     }
 }
 
+class CheckinSeeder extends Seeder
+{
+    public function run()
+    {
+        DB::table('check_ins')->delete();
+
+		$rooms = Room::all();
+		$items = Item::all();
+		srand(1);
+		foreach($rooms as $room) {
+			foreach($items as $item) {
+				for($i = 0; $i < rand(1,4); $i++) {
+					CheckIn::create([
+						'room_id'	=> $room->id,
+						'item_id'	=> $item->id,
+						'created_at' => rand(time()/2,time())
```

Comment:
We were looking at the DB table before we looked at the Seeder script... we had a chuckle when we realized where the dates came from. :smile: